timeseries/src/autogluon/timeseries/models/__init__.py

```diff
@@ -22,15 +29,22 @@
 )
 
 __all__ = [
```

Comment:
Can you please also add the new models to the following files?
- `docs/tutorials/timeseries/forecasting-model-zoo.md` - probably ZeroModel under "Baselines" & add a new section for sparse models
- add at least some models to `tests/smoketests/test_all_models.py`
- assign aliases & priorities in `src/autogluon/timeseries/models/presets.py`